index.html

```diff
@@ -869,7 +968,28 @@ <h3>Script Event</h3>
 
         <section>
           <h4>Text</h4>
-         <p>The <dfn>Text</dfn> object contains text content typically in a single language. This language may be the <a>Original</a> language or a <a>Translation</a> language.</p>
+          <p>The <dfn>Text</dfn> object contains text content typically in a single language.
+            This language may be the <a>Original</a> language or a <a>Translation</a> language.</p>
+          <p><a>Text</a> is defined as <dfn>Original</dfn> if it is any of:
+            <ol>
+              <li>the same language as the dialogue that it represents in the original programme audio;</li>
+              <li>a transcription of text visible in the programme video, in the same language as that text;</li>
+              <li>representative of non-dialogue sound;</li>
+              <li>a description of the scene in the programme video.</li>
+            </ol>
+          </p>
+          <p><a>Text</a> is defined as <dfn>Translation</dfn> if it is
+            a representation of an <a>Original</a> <a>Text</a> object in a different language.</p>
+          <p>The source language of <a>Translation</a> and, where applicable, <a>Original</a> <a>Text</a> objects
+            is indicated using the <a>Text Language Source</a> property.</p>
+          <aside class="note">The language of an <a>Original</a> <a>Text</a> object
+            can be different to the document's <a>Default Language</a>.</aside>
+          <aside class="note">The language of a <a>Text</a> object whose <a>Text Language Source</a> is <a>Original</a>
+            can be absent from the document's <a>Original Languages</a>.
+            For example a <a>transcript</a> of a short piece of dialogue spoken in a different language to the majority of the dialogue would be annotated
+            with both a different <code>xml:lang</code> value and as being <a>Original</a>,
```

Comment:
I'll check for this.

Reply:
I think this is okay, you may find some edge cases where it could be different if you look hard enough, but I'm not seeing anything that's inconsistent enough to make me want to change it.